app/components/ui/Navigation/Navigation.jsx

```diff
@@ -71,8 +71,10 @@ class Navigation extends React.Component {
             }
           </div>
           <div className={styles.mobileNavigation}>
-            <button type="button" className={styles.searchButton} onClick={this.toggleSecondarySearch} />
-            <button type="button" className={styles.hamburgerButton} onClick={toggleHamburgerMenu} />
+            {!isSFFamiliesSite()
+              && (<button type="button" className={styles.searchButton} onClick={this.toggleSecondarySearch} />)}
+            {!isSFFamiliesSite()
+              && (<button type="button" className={styles.hamburgerButton} onClick={toggleHamburgerMenu} />)}
```

Comment:
I think this would be clearer as a single `if` expression, rather than separate ones for each button, because otherwise it's not obvious upon first glance that the conditions are the same.

```suggestion
            {!isSFFamiliesSite() && (
              <>
                <button type="button" className={styles.searchButton} onClick={this.toggleSecondarySearch} />
                <button type="button" className={styles.hamburgerButton} onClick={toggleHamburgerMenu} />
              </>
            )}
```

Also, I noticed there was an existing `{!isSFFamiliesSite() && ` conditional expression just below these two changes, so could you combine them all into a single one? I think this just means putting the entire `styles.mobileNavigation` div element inside of the conditonal.

Reply:
Thanks! I was mostly being super lazy and didn't want to remember what react JSX errors meant. Added a Fragment to wrap the conditional.